superbench/benchmarks/model_benchmarks/pytorch_lstm.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the Pytorch LSTM model."""
+
+import time
+
+import torch
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Precision
+from superbench.benchmarks.model_benchmarks.model_base import Optimizer
+from superbench.benchmarks.model_benchmarks.pytorch_base import PytorchBase
+from superbench.benchmarks.model_benchmarks.random_dataset import TorchRandomDataset
+
+
+class LSTMBenchmarkModel(torch.nn.Module):
+    """The LSTM model for benchmarking."""
+    def __init__(self, input_size, hidden_size, num_layers, bidirectional, num_class):
```

Comment:
`num_class` -> `num_classes` to keep consistent

Reply:
Done.